sigma9.css

```diff
@@ -7,15 +7,17 @@
 
 @charset "utf-8";
 
+@import url('https://rsms.me/inter/inter.css');
+@import url('https://scp-wiki.wdfiles.com/local--files/component%3Atheme/Sans-Normalcy-v0.1.7.css');
 @import url('https://scp-wiki.wdfiles.com/local--files/component%3Atheme/font-bauhaus.css');
 @import url('https://fonts.googleapis.com/css2?family=Nanum+Gothic:wght@400;700');
 @import url('https://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css');
 
 /* COMMON */
 #content-wrap {
 	position: relative;
-	margin: 2em auto 0;
-	max-width: 1040px;
+	margin: 1.5em auto 0;
+	max-width: 1080px;
```

Comment:
Perhaps we should take this opportunity to convert all the `px` to `rem` instead?

Reply:
Since this is a big, breaking change, we may as well fit in necessary QoL fixes as part of it as well, instead of waiting for some future time to do it.